### PR TITLE
feat(fdc): Upgrade to v1beta endpoint

### PR DESCRIPTION
**Pull request by maneesht**

### CODE REVIEW

File: packages/firebase_data_connect/firebase_data_connect/lib/src/generated/connector_service.pbenum.dart [hunk 1]
```diff
@@ -1,7 +1,3 @@
-// Copyright 2024, the Chromium project authors.  Please see the AUTHORS file
```
Comment (russellwheatley) [inline packages/firebase_data_connect/firebase_data_connect/lib/src/generated/connector_service.pbenum.dart:1]: I appreciate this is quite annoying, but all the generated src code has lost their license headers.

**Review by russellwheatley — CHANGES_REQUESTED** [review]
The license headers + running `melos run format` for this PR 🙏

File: packages/firebase_data_connect/firebase_data_connect/lib/src/generated/connector_service.pbenum.dart (hunk 1 above)
Comment (maneesht) [reply packages/firebase_data_connect/firebase_data_connect/lib/src/generated/connector_service.pbenum.dart:1]: It looks like the melos scripts aren't actually adding the headers. I'll do it manually

Comment (maneesht) [reply packages/firebase_data_connect/firebase_data_connect/lib/src/generated/connector_service.pbenum.dart:1]: Updated

Comment (russellwheatley) [reply packages/firebase_data_connect/firebase_data_connect/lib/src/generated/connector_service.pbenum.dart:1]: just noticed this in the melos script: https://github.com/firebase/flutterfire/blob/main/melos.yaml#L285. Not sure if it needs to be there but this is why you're probably not seeing it added.

Comment (maneesht) [reply packages/firebase_data_connect/firebase_data_connect/lib/src/generated/connector_service.pbenum.dart:1]: Ah, makes sense

**Review by russellwheatley — APPROVED** [review]

File: packages/firebase_data_connect/firebase_data_connect/example/lib/generated/seed_movies.dart [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+  FirebaseDataConnect dataConnect;
+}
+
+class SeedMoviesTheMatrix {
+  late String id;
+
+  SeedMoviesTheMatrix.fromJson(Map<String, dynamic> json) : id = json['id'] {}
+
+  // TODO(mtewani): Fix up to create a map on the fly
```
Comment (cynthiajoan) [inline packages/firebase_data_connect/firebase_data_connect/example/lib/generated/seed_movies.dart:23]: I thought I see this removed in the generation code.

**Review by cynthiajoan — APPROVED** [review]

**Review by aashishpatil-g — APPROVED** [review]

**Pull request merged**